tests/test_csv_analysis.py

```diff
@@ -60,6 +61,31 @@ async def test_analyse_csv_big_file(setup_catalog, rmock, db, fake_check, produc
     assert profile["total_lines"] == expected_count
 
 
+@pytest.mark.slow
+async def test_exception_analysis(setup_catalog, rmock, db, fake_check, produce_mock):
+    """
+    Tests that exception resources (files that are too large to be normally processed) are indeed processed.
+    """
+    config.override(MAX_FILESIZE_ALLOWED=5000)
+    check = await fake_check(resource_id=config.LARGE_RESOURCES_EXCEPTIONS[0])
+    await db.execute(f"UPDATE catalog SET resource_id = '{config.LARGE_RESOURCES_EXCEPTIONS[0]}' WHERE id=1")
+    await db.execute("UPDATE catalog SET last_check = 1 WHERE id=1")
```

Comment:
If you UPDATE catalog with resource id before fake_check, you don't need to modify last_check, do you?

Reply:
Indeed, I'll change this!